packages/story-editor/src/components/publishModal/mainContent/storyPreview.js

```diff
@@ -16,20 +16,186 @@
  */
-import { Headline, THEME_CONSTANTS } from '@googleforcreators/design-system';
+import PropTypes from 'prop-types';
 import styled from 'styled-components';
+import {
+  Button,
+  Headline,
+  Icons,
+  Text,
+  THEME_CONSTANTS,
+  BUTTON_VARIANTS,
+  BUTTON_TYPES,
+  BUTTON_SIZES,
+} from '@googleforcreators/design-system';
+import { PAGE_RATIO } from '@googleforcreators/units';
 import { __ } from '@googleforcreators/i18n';
+/**
+ * Internal dependencies
+ */
+import { useConfig, useStory } from '../../../app';
 
+// Set the available space for cover preview image + overlay
 const PreviewContainer = styled.div`
-  margin: 8px 0;
-  object-fit: contain;
+  width: ${({ width }) => width}px;
+  height: ${({ height }) => height}px;
+  position: relative;
+  margin-top: 8px;
+`;
+PreviewContainer.propTypes = {
+  width: PropTypes.number.isRequired,
+  height: PropTypes.number.isRequired,
+};
+
+const PreviewWrapper = styled.div`
+  width: ${({ width }) => width}px;
+  height: ${({ height }) => height}px;
+  position: absolute;
+`;
+PreviewWrapper.propTypes = {
+  width: PropTypes.number.isRequired,
+  height: PropTypes.number.isRequired,
+};
+
+export const Image = styled.img`
+  height: 100%;
   width: 100%;
-  & > img {
-    border-radius: ${({ theme }) => theme.borders.radius.medium};
-  }
+  display: block;
+  aspect-ratio: auto ${PAGE_RATIO};
+  object-fit: cover;
+  border-radius: ${({ theme }) => theme.borders.radius.medium};
+  background: ${({ theme }) => theme.colors.interactiveBg.primaryNormal};
 `;
 
+const EmptyPlaceholder = styled.div`
+  height: 100%;
+  border-radius: ${({ theme }) => theme.borders.radius.medium};
+  background: ${({ theme }) => theme.colors.interactiveBg.primaryNormal};
+`;
+
+const Gradient = styled.div`
+  position: absolute;
+  bottom: 0;
+  height: 67%;
+  width: 100%;
+  border-radius: ${({ theme }) => theme.borders.radius.medium};
+  background: ${({ theme }) => theme.colors.gradient.posterOverlay};
+`;
+
+const ScrimWrapper = styled.div`
+  position: absolute;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  border-radius: ${({ theme }) => theme.borders.radius.medium};
+  background: ${({ theme }) => theme.colors.opacity.black3};
+`;
+
+const ScrimContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  overflow: hidden;
+`;
+
+const ScrimContent = styled.div`
+  height: 100%;
+  box-sizing: border-box;
+  position: absolute;
+  width: 100%;
+  top: 0;
+  padding: 8px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+`;
+
+const TopOverlay = styled.div`
+  display: grid;
+  grid-template-columns: 50% 50%;
+  grid-template-rows: auto;
+  grid-template-areas: 'publisherLogo editButton';
+  justify-content: inherit;
+  width: 100%;
+`;
+const PublisherLogo = styled.img`
+  grid-area: publisherLogo;
+  height: 24px;
+  width: 24px;
+`;
```

Comment:
AFAIK the publisher logo has some slight border radius

Reply:
you're right! will update.
<img width="358" alt="Screen Shot 2022-02-16 at 10 38 17 AM" src="https://user-images.githubusercontent.com/10720454/154323510-1c61d8fb-6df6-4603-aaf6-82fee988cb33.png">